processing/src/main/java/io/druid/segment/data/CompressedObjectStrategy.java

```diff
@@ -85,6 +92,22 @@ public Compressor getCompressor()
       {
         return UncompressedCompressor.defaultCompressor;
       }
+    },
+    /*
+    This value indicate no compression strategy should be used, and compression should not be block based
+     */
+    NONE((byte) 0xFE) {
+      @Override
+      public Decompressor getDecompressor()
+      {
+        return UncompressedDecompressor.defaultDecompressor;
```

Comment:
Nothing should call this, right? If not, let's do `throw new UnsupportedOperationException("No compression means no decompressor!")`